src/util/validationUtils.ts

```diff
@@ -48,3 +48,23 @@ export function validateParamIdSource(pathId: any, paramId: any) {
     );
   }
 }
+
+/**
+ * Checks that the content-type from the request headers accepts json + fhir.
+ */
+export function checkContentTypeHeader(contentType: string | undefined) {
```

Comment:
```suggestion
export function checkContentTypeHeader(contentType?: string) {
```
nitpicking here, but I think this is a bit more to our standard